### PR TITLE
Enable building Docker images for released packages

### PR DESCRIPTION
**Pull request by amarthadan**
Not really solving #1332 per se but rather a forgotten part of #1331. Also, some refactoring, clean up, and breaking into smaller parts that we can now build on :slightly_smiling_face:

### CODE REVIEW

**Review by Siegrift — COMMENTED** [review]
I tested this by building the images locally and testing them. I wanted to also use the local NPM packages but the current setup won't work on macOS due to [this](https://github.com/docker/for-mac/issues/2670#issuecomment-371249949).

I tried publishing ports for both `docker:scripts:npm-registry` command and also to the verdaccio container run inside the docker, but it didn't help

File: docker/README.md [hunk 1]
```diff
@@ -1,16 +1,13 @@
 # api3/airnode-packaging
 
-This is a Docker container for building all the packages in the monorepo, publishing the NPM packages (TODO) and
-building the Docker containers.
+This is a Docker container that can:
 
-## How it works
+- Start/stop a local NPM registry Docker container
+- Build and publish NPM packages to both local and official (TODO) NPM registry
```
[the remark this answers is not in the document]
Comment (amarthadan) [reply docker/README.md:6]: Yes, that one is intended, I'm working on that right now :slightly_smiling_face:

File: package.json [hunk 2]
```diff
@@ -45,9 +45,20 @@
     "dev:invoke": "(cd packages/airnode-node && yarn run dev:invoke)",
     "dev:list": "(cd packages/airnode-operation && yarn run dev:list)",
     "dev:stop": "(cd packages/airnode-operation && yarn run dev:stop)",
-    "docker:build": "yarn docker:build:packaging && yarn docker:build:images",
-    "docker:build:images": "docker run --rm -v $(pwd):/airnode -v /var/run/docker.sock:/var/run/docker.sock api3/airnode-packaging:latest",
-    "docker:build:packaging": "docker build --tag api3/airnode-packaging:latest --file docker/Dockerfile docker/",
+    "docker:build:local": "yarn docker:build:packaging && yarn docker:build:images:local",
```
[the remark this answers is not in the document]
Comment (amarthadan) [reply package.json:48]: I swear they were but I changed them so many times they got out of sync :sweat_smile: Will do.

**Comment by aquarat** [comment]
> I tested this by building the images locally and testing them. I wanted to also use the local NPM packages but the current setup won't work on macOS due to [this](https://github.com/docker/for-mac/issues/2670#issuecomment-371249949).

MacOS's lack of containers is... sad. I know it's generally slower due to the FS mounts but I didn't realise it also had this network limitation, it makes sense.

**Comment by Siegrift** [comment]
> MacOS's lack of containers is... sad. I know it's generally slower due to the FS mounts but I didn't realise it also had this network limitation, it makes sense.

Actually, the fix is simple (just publish the ports) and together with @amarthadan we already fixed the issue.

**Review by Siegrift — APPROVED** [review]
👍 LGTM

**Pull request merged**